app/views/govuk_publishing_components/components/_accordion.html.erb

```diff
@@ -22,44 +22,37 @@
   locales = {}
 
   data_attributes ||= {}
-  data_attributes[:module] = 'gem-accordion'
-  data_attributes[:anchor_navigation] = anchor_navigation
-
-  translations.each do |key, translation|
-    locales[key] = shared_helper.t_locale(translation)
-    data_attributes[key] = t(translation)
-  end
-
-  unique_locales = locales.values.uniq
-
-  if unique_locales.length > 1
-    data_attributes[:locale] = locales
-  else
-    if unique_locales[0] != I18n.locale
-      data_attributes[:locale] = unique_locales[0]
-    end
-  end
+  data_attributes[:module] = 'govuk-accordion'
 %>
 <% if items.any? %>
   <%= tag.div(class: accordion_classes, id: id, data: data_attributes) do %>
     <% items.each_with_index do |item, i| %>
       <%
+        # Nunjucks starts a loop on 1 and the client side JavaScript also
+        # adopts this behaviour. To prevent things from breaking, the index
+        # here also need to be increase by one. (Nunjucks is used by GOV.UK
+        # Frontend, which this component is based on.)
```

Comment:
Thanks for explaining this oddity although I found it a little confusing - I don't think it needs to mention Nunjucks. If my understanding is correct, would the following suit?

`The GOVUK Frontend JavaScript for this component uses a loop starting at 1 for accessing heading ID values, hence the increment below.`